tasks/offlinePackagingTasks.ts

```diff
@@ -58,7 +58,7 @@ function doPackageOffline() {
         new PlatformInformation('linux', 'x86_64')
     ];
 
-    let promise = Promise.resolve();
+    let promise: Promise<Result> = Promise.resolve<Result>(null);
```

Comment:
Does this need type on the right side even after you've specified the generic argument?